docs/guides/security/authentication/active_directory_authentication.md

```diff
@@ -183,3 +184,156 @@ To make this configuration change take effect, you must restart the
 ```sh
 [user@host ~]$ sudo systemctl restart sssd
 ```
+
+In the same way, if you don't want your home directories to be suffixed by the domain name, 
+you can add those options into your configuration file `/etc/sssd/sssd.conf`:
+
+```
+[domain/ad.company.local]
+use_fully_qualified_names = False
+override_homedir = /home/%u
+```
+
+Don't forget to restart the `sssd` service.
+
+### Restrict to certain users
+
+There are various methods to restrict access to the server to a limited list of users,
+ but this, as the name suggests, is certainly the simplest:
+
+Add those options into your configuration file `/etc/sssd/sssd.conf` and restart the service:
+
+```
+access_provider = simple
+simple_allow_groups = group1, group2
+simple_allow_users = user1, user2
+```
+
+Now, only users from group1 and group2, or user1 and user2 will be able to connect to the server using sssd!
+
+## Interact with the AD using `adcli`
+
+`adcli` is a CLI to perform actions on an Active Directory domain.
+
+- If not yet installed, install the required package
+
+```sh
+[user@host ~]$ sudo dnf install adcli
+```
+
+- Test if you have ever joigned an Active Directory domain:
+
+```sh
+[user@host ~]$ sudo adcli testjoin
+Successfully validated join to domain ad.company.local
+```
+
+- Get more advanced informations about the domain:
+
+```sh
+[user@host ~]$ adcli info ad.company.local
+[domain]
+domain-name = ad.company.local
+domain-short = AD
+domain-forest = ad.company.local
+domain-controller = dc1.ad.company.local
+domain-controller-site = site1
+domain-controller-flags = gc ldap ds kdc timeserv closest writable full-secret ads-web
+domain-controller-usable = yes
+domain-controllers = dc1.ad.company.local dc2.ad.company.local
+[computer]
+computer-site = site1
+```
+
+- More than a consulting tool, you can use adcli to interact with your domain: manage users or groups, change password, etc. 
+
+Example: use `adcli` to get information about a computer:
+
+!!! Note
+
+    This time we will provide an admin username thanks to the `-U` option
+
+```sh
+[user@host ~]$ adcli show-computer pctest -U admin_username
+Password for admin_username@AD: 
+sAMAccountName:
+ pctest$
+userPrincipalName:
+ - not set -
+msDS-KeyVersionNumber:
+ 9
+msDS-supportedEncryptionTypes:
+ 24
+dNSHostName:
+ pctest.ad.company.local
+servicePrincipalName:
+ RestrictedKrbHost/pctest.ad.company.local
+ RestrictedKrbHost/pctest
+ host/pctest.ad.company.local
+ host/pctest
+operatingSystem:
+ Rocky
+operatingSystemVersion:
+ 8
+operatingSystemServicePack:
+ - not set -
+pwdLastSet:
+ 133189248188488832
+userAccountControl:
+ 69632
+description:
+ - not set -
+```
+
+Example: use `adcli` to change user's password:
+
+```sh
+[user@host ~]$ adcli passwd-user user_test -U admin_username
+Password for admin_username@AD: 
+Password for user_test: 
+[user@host ~]$ 
+```
+
+## Troubleshooting
+
+Sometimes, the network service will start after sssd, that cause trouble with authentication.
```

Comment:
use of sssd is inconsistent in this doc, sometimes expressed as SSSD. Unless there is a reason, it's probably worth a find and replace to make them all the same.